tests/pathfinding/test_fee_schedule.py

```diff
@@ -74,7 +74,8 @@ def __init__(self, channels: List[dict]):
         for _, _, cv in self.G.edges(data="view"):
             cv.capacity = 100
 
-        # make nodes reachable
+        # create reachability mapping for testing
+        self.address_to_reachability: Dict[Address, AddressReachability] = {}
         for node in self.G.nodes:
             self.address_to_reachability[node] = AddressReachability.REACHABLE
```

Comment:
Now that we construct the dict here, we could use a dict comprehension
```python
self.address_to_reachability: Dict[Address, AddressReachability] = {
    node: AddressReachability.REACHABLE
    for node in self.G.nodes
}
```
Not much of a difference, but a bit easier to read IMO.